### PR TITLE
Add axes labels to PD plots

### PR DESCRIPTION
**Pull request by jameswex**
* Motivation for features / changes

Add axes labels to PD plots - X and Y labels for line charts, only  Y labels for bar charts as the bar labels themselves are self-explanatory.

* Technical description of changes

Put PD plots in a new wrapping div that contains the axes labels as well. When we delete plots we now do it by deleting this wrapping div.

* Screenshots of UI changes

pdlabels

* Detailed steps to verify changes work correctly (as executed by you)

Ran demos, opened and removed charts to see them added and removed. Verified correct label visuals. Manually tested very long labels to ensure correct handling of long strings.

* Alternate designs / implementations considered

### CODE REVIEW

**Comment by jameswex** [comment]
@tolga-b please review

**Comment by tolga-b** [comment]
Thank you James. Checking your visuals, I think you tested this on age regression demo. It seems a little confusing to say "Inference score" when it is regressing a variable. Would it be possible to add an if else clause for that?

**Comment by jameswex** [comment]
Great point Tolga, fixed. Tested regression and classification

File: tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html [hunk 1]
```diff
@@ -5007,7 +5043,19 @@ <h2>Show similarity to selected datapoint</h2>
         chart.seriesWithoutTooltips = thresholdSeries;
         chart.colorScale.domain(allSeries);
         chart.setVisibleSeries(allSeries);
-        return chart;
+        holder.appendChild(chart);
+
+        const yLabel = document.createElement('div');
+        yLabel.classList.add('pd-y-label');
+        yLabel.innerText = this.isRegression_(this.modelType) ?
+          'Inference value' : 'Inference score';
```
Comment (stephanwlee) [inline tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: +2 whitespace if you are following Google JS Style guide.

File: tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html [hunk 2]
```diff
@@ -1151,6 +1151,31 @@
...
+        bottom: 0;
+        color: #5f6368;
+        font-size: 12px;
+        overflow: hidden;
+        padding: 0 0 0 40px;
+        position: absolute;
+        text-align: center;
+        text-overflow: ellipsis;
+        width: 100%;
+        white-space: nowrap;
```
Comment (stephanwlee) [inline tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: in the screenshot, I see wrapping of text in the x-axis. Is this referring to some other element?

File: tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html (hunk 1 above)
Comment (jameswex) [reply tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: done in both places

File: tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html (hunk 2 above)
Comment (jameswex) [reply tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: its for wrapping of the axes label (only showing on line charts). so in the screen shot it is "capital-loss". i checked with a very long string locally and showed correct ellipsis and non-wrapping behavior.

Comment (stephanwlee) [reply tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: Ah, got it. Thanks!

**Pull request merged**